iocage_lib/ioc_json.py

```diff
@@ -1207,6 +1213,12 @@ def check_jail_config(self, conf):
             # New jail creation
             pass
 
+        # Version 24 keys
```

Comment:
Version `26` ;)
Also i don't like this personally for the reasons you pointed out earlier ( user explicitly setting this and on another bump this gets removed again resulting in more confusion ) but we will stick with it for now. I'll give higher priority to #963 now.